documentation/smg/10_how_to_use_the_SAN_area.md

```diff
@@ -0,0 +1,54 @@
+# How to use the SAN area
+
+ISIS has provided a storage area in which PACE can keep large test data. The
+area should contain data files from which we derive smaller sets of data
+(alongside scripts recording how the smaller sets were generated) and files
+that are too large to store in git repositories.
+
+A FedID is required in order to mount the SAN area. In order to mount the SAN
+on PACE's systems, a dedicated user ID with which to mount the SAN area has
+been provided to PACE.
+
+## Mounting the SAN on Jenkins
+
+The path to the SAN is stored as a secret in Jenkins with the ID `SAN_path`.
+To mount the SAN when running a Jenkins pipeline, you should use the dedicated
+user ID provided to PACE. The credentials for this are stored in a credentials file in Jenkins the with ID `SAN_credentials_file`,
+see https://www.jenkins.io/doc/book/using/using-credentials/ for an overview
```

Comment:
Now the path and credentials are stored as jenkins secrets, `see https://www.jenkins.io/doc/book/using/using-credentials/` doesn't fit here as includes an overview of both the credentials file and the secret path.

It makes more sense to move this link after the description of the file format or the the start. That way the section becomes more structured as:
- everything is stored in jenkins secrets
- here a link to how to use jenkins secrets 
- the path is < here >
- the credentials are < here > in < this > format